### DISCUSSION TITLE
Review Changes button not working as of about a week ago

### DISCUSSION THREAD

**Discussion by SergeyBarskiyTyler** [community/community#78668] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

If I go to a pull request, click add review, then click on ""Review Changes" button, nothing happens.  Same with "Finish my review" button.  "I have to reload the page at that point to complete the review.  This is a recent bug, maybe as of a week ago,

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for reporting - I've made sure this report has gotten passed along to our team 🤠 .
